packages/lesswrong/components/peopleDirectory/PeopleDirectoryFilters.tsx

```diff
@@ -69,21 +75,49 @@ export const PeopleDirectoryFilters = ({classes}: {
       </div>
       <div className={classes.options}>
         {view === "list" &&
-          <PeopleDirectoryFilterDropdown
-            title="Columns"
-            icon="ViewColumns"
-            style="button"
-            className={classes.columnsList}
-          >
-            {columns.filter(({hideable}) => hideable).map((state) => (
-              <PeopleDirectorySelectOption state={state} key={state.value} />
-            ))}
-            {columnsEdited &&
-              <div>
-                <PeopleDirectoryClearAll text="Reset" onClear={resetColumns} />
-              </div>
-            }
-          </PeopleDirectoryFilterDropdown>
+          <>
+            <PeopleDirectoryFilterDropdown
+              title="Sort"
+              icon="ArrowsUpDown"
+              style="button"
+              className={classes.columnsList}
+            >
+              <PeopleDirectoryCheckOption
+                label="Default"
+                selected={isDefaultSorting}
+                onSelect={setSorting.bind(null, null)}
+              />
+              {columns.map(({label, sortField, defaultSort}) => sortField
+                ? (
+                  <PeopleDirectoryCheckOption
+                    key={label}
+                    label={label}
+                    selected={sortField === sorting.field}
+                    onSelect={setSorting.bind(null, {
+                      field: sortField,
+                      direction: defaultSort ?? "asc",
+                    })}
```

Comment:
I would like clicking an already checked option to switch back to the default sorting